djangoProject/user/views.py

```diff
@@ -1,6 +1,23 @@
 from django.shortcuts import render
-from django.http import HttpResponse
+from django.http import HttpResponse, JsonResponse
+from django.core import serializers
+
+from user.models import User
+
+
+def users(request):
+    users = User.objects.values()
+    data = list(users)
+    return JsonResponse({'data': data})
+
+
+def users(request):
+    users = User.objects.values()
+    data = list(users)
+    return JsonResponse(data, safe=False, status=200)
 
 
 def users(request):
-    return HttpResponse("Hello, users!")
+    qs = User.objects.all()
+    qs_json = serializers.serialize('json', qs)
```

Comment:
it is a preferable way however model serializer should be used. I will talk about it in context of DRF